database/migrations/000062_user_invites.up.sql

```diff
@@ -0,0 +1,30 @@
+-- Copyright 2024 Stacklok, Inc
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--      http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+BEGIN;
+
+CREATE TABLE IF NOT EXISTS user_invites (
+    code          TEXT NOT NULL PRIMARY KEY,
+    email         TEXT NOT NULL,
```

Comment:
Removing the index on `(email, project)` means that we no longer have an index on email.  I'd be tempted to re-add this.

Reply:
Ugh, you're right! I must have deleted it along the other stuff 🤦‍♂️

Reply:
I pointed out that a `UNIQUE(email, project)` index would prevent inviting someone to multiple roles in a project, so it was sort of my fault.